region/maridia/inner-yellow.json

```diff
@@ -907,8 +939,79 @@
                     "Then morph on top and damage boost into the morph tunnel as it unfreezes."
                   ]
                 }
-              ],
-              "devNote": "FIXME Menus are tricky to deal with with Power Beam alone, information should be added to kill or avoid them, likely while adding a third node."
+              ]
+            }
+          ]
+        },
+        {
+          "from": 3,
+          "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "HiJump",
+                      "SpaceJump",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies",
+                      "canPreciseWalljump"
+                    ]}
+                  ],
+                  "devNote": "The menus don't really matter with these strats."
+                },
+                {
+                  "name": "Kill the Menus",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "h_canIBJ",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir"
+                    ]},
+                    {"or":[
+                      {"enemyKill":{
+                        "enemies": [ [ "Menu", "Menu", "Menu" ] ],
+                        "explicitWeapons": [
+                          "PowerBomb",
+                          "ScrewAttack",
+                          "Wave",
+                          "Spazer",
+                          "Plasma"
+                        ]
+                      }},
+                      {"and":[
+                        {"enemyKill":{ "enemies": [ [ "Menu", "Menu", "Menu" ] ] }},
+                        {"or":[
+                          {"and":[
+                            "canDodgeWhileShooting",
+                            "canTrickyJump"
+                          ]},
+                          {"and":[
+                            "canDodgeWhileShooting",
+                            {"enemyDamage": { "enemy": "Menu", "type": "contact", "hits": 1 }}
+                          ]},
+                          {"enemyDamage": { "enemy": "Menu", "type": "contact", "hits": 2 }}
+                        ]}
+                      ]}
+                    ]}
+                  ],
+                  "devNote": "The Menus are relatively hard to dodge, and would end up with the same requirements as killing them with the Power Beam."
+                }
+              ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [ "Morph" ]
```

Comment:
Does this need something for dealing with Menus?
In testing I was blocked by left menu, but never in runs.
Right menu I always kill with a missile, but it looks like you can wait a second for it to pass.

Reply:
I think it is simple enough to be techless. There are probably strats of this difficulty all over. 
The first menu can be jumped over, or lured and then avoided or killed pretty easily.
The right menu can be jumped over, or killed from the left water patch.